src/main/java/uk/sky/cqlmigrate/CassandraLockConfig.java

```diff
@@ -21,6 +25,8 @@ public static CassandraLockConfigBuilder builder() {
 
     public static class CassandraLockConfigBuilder extends LockConfig.LockConfigBuilder {
 
+        private ConsistencyLevel consistencyLevel = ConsistencyLevel.LOCAL_ONE;
```

Comment:
I don't think this default is tested, can you add a test for it?

Reply:
Hi @adamdougal, I have added some test coverage